src/main.rs

```diff
@@ -560,6 +599,22 @@ async fn launch_rocket(pool: db::DbPool, extra_debug: bool) -> Result<(), Error>
         CONFIG.shutdown();
     });
 
+    #[cfg(unix)]
+    {
+        tokio::spawn(async move {
+            let mut signal_user1 = tokio::signal::unix::signal(SignalKind::user_defined1()).unwrap();
+            loop {
+                // If we need more singles to act upon, we might want to use select! here.
```

Comment:
I think this is what you mean right?
```suggestion
                // If we need more signals to act upon, we might want to use select! here.
```

Reply:
Unless i would want to sing less 🤣.
I really need to check spelling before i commit haha.